http4s/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/CollectorService.scala

```diff
@@ -173,11 +179,19 @@ class CollectorService[F[_]: Sync](
     e
   }
 
-  // TODO: Handle necessary cases to build http response in here
   def buildHttpResponse(
+    queryParams: Map[String, String],
     headers: Headers,
+    redirect: Boolean,
     pixelExpected: Boolean
   ): Response[F] =
+    if (redirect)
```

Comment:
`redirect` is used in [here](https://github.com/snowplow/stream-collector/blob/master/core/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/CollectorService.scala#L125) as well in the old collector. Since we don't have cookie bounce feature in the new collector, it is used only in one place. Therefore, I think it would be better if we leave as it is.